edisgo/grid/components.py

```diff
@@ -254,6 +257,65 @@ def peak_load(self):
 
         return peak_load
 
+    @property
+    def reactive_power_mode(self):
+        """
+        Power factor mode of Load.
+
+        If the power factor is set, then it is necessary to know whether the
+        it is leading or lagging. In other words this information is necessary
+        to make the generator behave in an inductive or capacitive manner.
+        Essentially this changes the sign of the reactive power Q.
+
+        The convention used here in a load is that:
+        - when `reactive_power_mode` is 'inductive' then Q is positive
+        - when `reactive_power_mode` is 'capacitive' then Q is negative
+
+        In the case that this attribute is not set, it is retrieved from the
+        network config object depending on the voltage level the load is in.
+
+        Returns
+        -------
+        :obj: `str` : Power factor mode
+            Either 'inductive' or 'capacitive'
+        """
+        if self._reactive_power_mode is None:
+            if isinstance(self.grid, MVGrid):
+                self._reactive_power_mode = self.grid.network.config[
+                    'reactive_power_mode']['mv_load']
+            elif isinstance(self.grid, LVGrid):
+                self._reactive_power_mode = self.grid.network.config[
+                    'reactive_power_mode']['lv_load']
+
+        return self._reactive_power_mode[1:-1]
```

Comment:
Why is this necessary?

Reply:
I mean [1:-1]

Reply:
When 'inductive' and 'capacitive' are provided as strings in the config file, the first and the last character seems to be some non-readable characters. May be I could handle this better by directly influencing the parsing.

Reply:
I just found out that this is caused by using the `'`..`'` symbols in the config files. I have removed these and also the [1:-1].